rio_tiler/utils.py

```diff
@@ -36,6 +36,7 @@ def get_array_statistics(
     categorical: bool = False,
     categories: Optional[List[float]] = None,
     percentiles: Optional[List[int]] = None,
+    coverage: Optional[numpy.ndarray] = None,
```

Comment:
Since I like really like typing 🙈 maybe
```py
from numpy.typing import NDArray
...
coverage: Optional[NDArray[np.floating]] = None
```
I think from reading this, it's expected to be a float? Even if it doesn't impact type checking much, I always find it helpful to recall what data types of arrays are expected where